packages/spatial-id-svc-area/src/client-blocked-areas.ts

```diff
@@ -322,6 +323,11 @@ export const getBlockedAreas = async function* ({
     payload,
     abortSignal,
   })) {
+    if (chunk.result.objects[0].objectId !== '0') {
+      objectId = chunk.result.objects[0].objectId;
```

Comment:
Isnt it better to do optional chaining? using ?. or else if data wasn't present it could lead to errors